NuGet/dotnet-pack.bat

```diff
@@ -31,7 +31,7 @@ dotnet pack ../Source/Extensions/Blazorise.QRCode/Blazorise.QRCode.csproj -o . -
 dotnet pack ../Source/Extensions/Blazorise.RichTextEdit/Blazorise.RichTextEdit.csproj -o . -c Release -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
 dotnet pack ../Source/Extensions/Blazorise.Sidebar/Blazorise.Sidebar.csproj -o . -c Release -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
 dotnet pack ../Source/Extensions/Blazorise.SignaturePad/Blazorise.SignaturePad.csproj -o . -c Release -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
-dotnet pack ../Source/Extensions/Blazorise.Snackbar/Blazorise.Snackbar.csproj -o . -c Release -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
+dotnet pack ../Source/Extensions/Blazorise.Snackbar/Blazorise.Snackbar.csproj -o . -c Release -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg -p:IsPack=true
```

Comment:
Do we still need IsPack if Fosy is now used?

Reply:
This question has 2 layers.
- Yes - we need to distinguish between packing and simple build. I updated docs in last commit for explanation.
- I wish there was something like `IsPack` automatically added, so we wouldn't  need our custom `IsPack`, but so far I haven't found it.

Reply:
Why do we need to know simple build from publish? We need to remove all SG references at all times.